.github/workflows/test.yml

```
@@ -85,6 +85,8 @@ jobs:
           java-version: '11'
           cache: 'sbt'
       - name: Integration Tests
+        if: github.event_name != 'pull_request' ||
```

```suggestion
        # This is here instead of on the whole job because, if a job is skipped, so are dependent jobs.
        # If this job were skipped, all_tests_passed would be skipped too.
        # By having this "if" here, this job returns success so that all_tests_passed will succeed too.
        if: github.event_name != 'pull_request' ||
```
Useful to have a comment describing why this is here for posterity. I copied this from FIRRTL but edited it slightly.